pages/index.tsx

```diff
@@ -41,31 +43,31 @@ const HomePage: React.FC<InferGetStaticPropsType<GetStaticProps>> = ({
       <ErrorBanner errors={errors} />
       <Grid hasGutter>
         <GridItem xl2={3} xl={4} lg={5} md={6} sm={12} span={12}>
-          <DevfileFilter
+          <DevfileGalleryFilter
             tagsStateWithFreq={tagsStateWithFreq}
             typesStateWithFreq={typesStateWithFreq}
             setTagsStateWithFreq={setTagsStateWithFreq}
             setTypesStateWithFreq={setTypesStateWithFreq}
           />
         </GridItem>
         <GridItem xl2={9} xl={8} lg={7} md={6} sm={12} span={12}>
-          <DevfileSearchBar
+          <DevfileGallerySearchBar
             devfileCount={filteredDevfiles.length}
             onSearchBarChange={onSearchBarChange}
             searchBarValue={searchBarValue}
           />
-          <DevfileGrid devfiles={filteredDevfiles} />
+          <DevfileGalleryGrid devfiles={filteredDevfiles} sourceRepos={sourceRepos} />
         </GridItem>
       </Grid>
     </>
   );
 };
 
-const isSearchBarValueIn = (value: string | undefined, searchBarValue: string) =>
-  value?.toLowerCase().includes(searchBarValue.toLowerCase());
+const isSearchBarValueIn = (value: string | undefined, searchBarValue: string): boolean =>
+  !!value?.toLowerCase().includes(searchBarValue.toLowerCase());
```

Comment:
Double negation is used more commonly in JS than other languages because of instances like this. If value is `null` and doesn't have a string it cannot be turned into a lower case; therefore, the function would return `undefined` which is falsey. This is/would be perfectly valid js; however, since I turned on the return types in ts we should really only return one type. This is why the double negation is here.